src/app/wallet/v2/views/utils.nim

```diff
@@ -0,0 +1,45 @@
+import atomics, strformat, strutils, sequtils, json, std/wrapnils, parseUtils, tables, chronicles, web3/[ethtypes, conversions], stint
+import NimQml, json, sequtils, chronicles, strutils, strformat, json
+
+logScope:
+  topics = "utils-view"
```

Comment:
nit: align naming between qml and nim: Use settings in both cases?

Reply:
hmmm 🧐 makes sense!

Reply:
Done